### ISSUE TITLE
prettier workflow error

### ISSUE THREAD

**Issue by citruscai** [MLH-Fellowship/orientation-project-js-24.FAL.B#17]
whenever prettier runs, prettier says its not installed

**Comment by dnhan1707** [comment]
Then I believe you have to install it first
What I did was:
`npm install -g prettier
`
and then run
`yarn prettier --write .
`
if yarn is not installed you can install it too `npm install -g yarn`

**Comment by vantage-ola** [comment]
> Then I believe you have to install it first What I did was: `npm install -g prettier ` and then run `yarn prettier --write . ` if yarn is not installed you can install it too `npm install -g yarn`

yes, that solves the issue. @citruscai  do you still have issues with prettier? if not? you can close the issue.
